KVStore.md

```diff
@@ -0,0 +1,104 @@
+# KVStore <!-- omit in toc -->
```

Comment:
I did not review the README, assuming its just a copy & paste.

Reply:
No this is a new `README.md` there wasnt one before (or I missed it) however its pretty basic